ContactManager/ContactManager/Controller/AddNewContactViewController.swift

```diff
@@ -0,0 +1,169 @@
+//
+//  AddNewContactViewController.swift
+//  ContactManager
+//
+//  Created by 전성수 on 10/11/23.
+//
+
+import UIKit
+
+protocol SendPersonContactData {
+    func sendData(name: String, age: String, digits: String)
+}
+
+final class AddNewContactViewController: UIViewController {
+    
+    var delegate: SendPersonContactData?
+    
+    @IBOutlet weak var inputName: UITextField!
+    @IBOutlet weak var inputAge: UITextField!
+    @IBOutlet weak var inputDigits: UITextField!
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        inputDigits.delegate = self
+    }
+    
+    @IBAction private func goBack(_ sender: Any) {
+        presentUserCancelConfirmAlert()
+    }
+    
+    @IBAction func saveNewPersonContact(_ sender: Any) {
+        guard var name = inputName.text, let age = inputAge.text, let digits = inputDigits.text
+        else { return }
```

Comment:
guard 문에서는 예외처리를 return 부분을 개행해서 잘보이게 처리하는게 좋습니다 참고하세요!

```suggestion
        guard var name = inputName.text, let age = inputAge.text, let digits = inputDigits.text else {
             return
        }

Reply:
여기서 return 예외처리에는 alert가 따로 처리가 되지않네요. 이 부분도 한번 확인 부탁드립니다!

Reply:
[bca10a1](https://github.com/tasty-code/ios-contact-manager-ui/pull/36/commits/bca10a1666bf621d9ae120495c03023fcf54077f)
```return presentInputValidationAlert(InputError.exception)```으로 alert가 처리되도록 하였습니다.